### PR TITLE
workflows

### PR DESCRIPTION
**Pull request by daleyp24**
## JIRA Ticket
Ticket: https://ihm-it.atlassian.net/browse/IHRACP-5340

Action: update readme

#### Major Changes:
update readme
Better shalls 

## How To Test
just read it

### CODE REVIEW

**Comment by daleyp24** [comment]
Do we want GHA in this ticket scope. @IanNoyes

**Comment by daleyp24** [comment]
need to update readme

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
```
Comment (KMichaelMcGaw) [inline .github/workflows/release-drafter.yml:1]: Is the idea behind the release drafter that we draft a release and then deploy with the tag being the released version? 
Would it be a better idea to hold this off until we CI/CD this out?

Comment (daleyp24) [reply .github/workflows/release-drafter.yml:1]: I think it i good to have it here and does not hurt anything. keeps us in line code artifact and lowers confusion

**Review by KMichaelMcGaw — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,27 +1,64 @@
...
+- Make sure you have follow the steps
+[here](https://github.com/iheartradio/content-platform-documentation/blob/master/private_python_modules/README.md)
+so you can download our private repositories.
+
+Before the first run of the service::
+
+    $ pip install pip-tools
+    $ pip install -r requirements-dev.txt
+    $ pre-commit install
+    $ pip-compile requirements-public.in
```
Comment (poonamshinde-ihm) [inline README.md]: we don't have requirements-public and requirements-private files this needs to update with requirements.in and requirements.txt

Comment (daleyp24) [reply README.md]: done

**Review by poonamshinde-ihm — APPROVED** [review]

**Pull request merged**